packages/hub/src/app/users/[username]/models/[slug]/ModelLayout.tsx

```diff
@@ -96,8 +106,32 @@ function useFixModelUrlCasing(model: ModelLayoutQuery$data["model"]) {
 
 type Props = PropsWithChildren<CommonProps>;
 
+export const EntityNodes = (
```

Comment:
Um, I guess you did global `entityNode` -> `EntityNode` find-and-replace?

Highly recommend `Rename symbol` in VS Code for these refactorings (`fn+f2`), it always does the right thing (looks up all references in other files).

Reply:
Oh, thanks!